examples/st-device-app/android/App/core/data/src/test/java/com/samsung/matter/chipstdeviceapp/core/data/ExampleUnitTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.samsung.matter.chipstdeviceapp.core.data
+
+import org.junit.Assert.*
+import org.junit.Test
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
Are these used/exercised anywhere? The same example seems to be copied and pasted in many places. These files do not seem useful (unless they are required for some build to succeed I guess).

Reply:
This is our baseline commit. We will add follow up commit from our implementation. We will use this for Next Unit Test related patch.